[2025-10-31T02:52:02.141+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T02:52:02.169+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:32:27.281802+00:00 [queued]>
[2025-10-31T02:52:02.178+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:32:27.281802+00:00 [queued]>
[2025-10-31T02:52:02.179+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-10-31T02:52:02.195+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-10-31 02:32:27.281802+00:00
[2025-10-31T02:52:02.200+0000] {standard_task_runner.py:63} INFO - Started process 348 to run task
[2025-10-31T02:52:02.204+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgres_dag', 'load_to_postgres', 'manual__2025-10-31T02:32:27.281802+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/api_to_postgres_dag.py', '--cfg-path', '/tmp/tmpnc4pgmsn']
[2025-10-31T02:52:02.207+0000] {standard_task_runner.py:91} INFO - Job 70: Subtask load_to_postgres
[2025-10-31T02:52:02.269+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:32:27.281802+00:00 [running]> on host 754154eb0275
[2025-10-31T02:52:02.364+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lenny' AIRFLOW_CTX_DAG_ID='api_to_postgres_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T02:32:27.281802+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T02:32:27.281802+00:00'
[2025-10-31T02:52:02.365+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T02:52:02.413+0000] {logging_mixin.py:188} INFO - âœ… Loaded data into PostgreSQL table.
[2025-10-31T02:52:02.414+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-31T02:52:02.415+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T02:52:02.427+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_postgres_dag, task_id=load_to_postgres, run_id=manual__2025-10-31T02:32:27.281802+00:00, execution_date=20251031T023227, start_date=20251031T025202, end_date=20251031T025202
[2025-10-31T02:52:02.457+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-31T02:52:02.475+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-31T02:52:02.478+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
